src/openseadragon.js

```diff
@@ -187,6 +187,9 @@
   * @property {Number} [opacity=1]
   *     Default opacity of the tiled images (1=opaque, 0=transparent)
   *
+  * @property {Boolean} [preload=false]
+  *     If true, tiles in image load without drawing. Defaults to false, so tiles are loaded and drawn.
```

Comment:
I think the behavior should be that `preload` causes tiles to load even on 0 opacity images, but that it doesn't affect whether the tiles draw. This doc comment makes it sound like `preload: true` will make the tiles not draw; please update.